<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc.qrc">
    <normaloff>:/new/prefix1/images/code.png</normaloff>:/new/prefix1/images/code.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget
{
background-color: rgb(235, 243, 255);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="Edit_Frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
       <widget class="QFrame" name="msg_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="command_Frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionHighlightKeyWords"/>
    <addaction name="actionFont"/>
    <addaction name="actioncomment_"/>
    <addaction name="actionUncomment_"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>通讯设置(S)</string>
    </property>
    <addaction name="actionOpenSerial"/>
    <addaction name="actionCloseSerial"/>
    <addaction name="actionSerialSettings"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>构建(B)</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionDownload"/>
    <addaction name="actionFlash"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionContactUs"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(T)</string>
    </property>
    <addaction name="actionImageToBin"/>
    <addaction name="actionBurnConfig"/>
    <addaction name="actionUpgrade"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_T"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 243, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/open.png</normaloff>:/new/prefix1/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/save.png</normaloff>:/new/prefix1/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/exit.png</normaloff>:/new/prefix1/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/saveAs.png</normaloff>:/new/prefix1/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionHighlightKeyWords">
   <property name="text">
    <string>语法高亮设置</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actionOpenSerial">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/connect.png</normaloff>:/new/prefix1/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="actionCloseSerial">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/disconnect.png</normaloff>:/new/prefix1/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="actionSerialSettings">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/setting.png</normaloff>:/new/prefix1/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>手动设置</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/download.png</normaloff>:/new/prefix1/images/download.png</iconset>
   </property>
   <property name="text">
    <string>下载调试</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/compile.png</normaloff>:/new/prefix1/images/compile.png</iconset>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionContactUs">
   <property name="text">
    <string>联系我们</string>
   </property>
  </action>
  <action name="actionFlash">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/burn.png</normaloff>:/new/prefix1/images/burn.png</iconset>
   </property>
   <property name="text">
    <string>烧录</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>作者</string>
   </property>
  </action>
  <action name="actionImageToBin">
   <property name="text">
    <string>ImageToBin</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>comment</string>
   </property>
  </action>
  <action name="actionUncomment">
   <property name="text">
    <string>uncomment</string>
   </property>
  </action>
  <action name="actioncomment_">
   <property name="text">
    <string>注释选择</string>
   </property>
  </action>
  <action name="actionUncomment_">
   <property name="text">
    <string>去掉注释</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/new.png</normaloff>:/new/prefix1/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionBurnConfig">
   <property name="text">
    <string>烧录配置</string>
   </property>
  </action>
  <action name="actionUpgrade">
   <property name="text">
    <string>升级固件</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/new/prefix1/images/clear.png</normaloff>:/new/prefix1/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
